src/core/localStorageService.ts

```diff
@@ -0,0 +1,31 @@
+export default class LocalStorageService<JSON> {
+  #key: string
+  #storage: Storage
+
+  constructor(key: string) {
+    this.#storage = window.localStorage
+    this.#key = key
+  }
+
+  get(): JSON | null {
+    const value = this.#storage.getItem(this.#key)
+    if (value === undefined || value === null || value === '') return null
+    return this.toTargetType(value)
+  }
+
+  set(newValue: JSON) {
+    this.#storage.setItem(this.#key, this.toString(newValue))
+  }
+
+  clean() {
+    this.#storage.removeItem(this.#key)
+  }
+
+  toTargetType(data: any): JSON {
+    return JSON.parse(data)
+  }
+
+  toString(data: JSON): string {
+    return JSON.stringify(data)
+  }
```

Comment:
```suggestion
 stringfy(data: JSON): string {
    return JSON.stringify(data)
  }
```
toString은 객체 내장메소드가 생각나서 stringfy와 같이 다른이름이 나은것 같습니다.

Reply:
toTargetType / toString 의 두운을 맞춘건데, 앞서 언급하신대로 JSON 형변환만 하고 있는 현상황에는 굳이 이 메서드가 존재할 필요가 없겠습니다.